screens/signUp.js

```diff
@@ -1,11 +1,16 @@
+/* eslint-disable no-unused-vars */
```

Comment:
What is this being used for? Ideally, we should not have any variables that are not being used.